test/client/fixtures/sandbox/event/focus-blur-change-test.js

```diff
@@ -880,3 +880,50 @@ asyncTest('error on the http://phonejs.devexpress.com/Demos/?url=KitchenSink&sm=
         startNext();
     });
 });
+
+asyncTest('Should restore scrolling elements with overflow hidden after focus (GH-221)', function () {
```

Comment:
> We use the direct word order in client tests since there's no it function. don't we?

correct

Mocha tests has BDD-style naming, while in qunit we use traditional TDD naming